compiler/bcq-tools/README.md

```diff
@@ -0,0 +1,44 @@
+# BCQ Tools
+
+This directory includes some tools related with BCQ.
+
+## preserve_bcq_info
+
+### Purpose
+
+`preserve_bcq_info` is for preserving constant nodes which are including BCQ information.
+When `.pb` file is converted to `.tflite` file by TFlite converter, constant nodes whose values are exactly same are removed and then linked to only one representative node.
+It causes information missing problem because we don't know which constant nodes should be linked to even we still want to apply BCQ.
```

Comment:
maybe...
`This causes lost of information problem as we don't know which constant node should be linked to the node that we want to apply BCQ`